From e3325c534934d14b0d87fed9d8da3bfa32683a8e Mon Sep 17 00:00:00 2001
From: Josh Wu <josh.wu@atmel.com>
Date: Tue, 30 Sep 2014 10:54:47 +0800
Subject: [PATCH] ARM: at91: switch to generic board for at91sam9260 series
 boards

Includes at91sam9260ek, at91sam9g20ek and at91sam9xeek boards.

Signed-off-by: Josh Wu <josh.wu@atmel.com>
---
 include/configs/at91sam9260ek.h | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/include/configs/at91sam9260ek.h b/include/configs/at91sam9260ek.h
index 73917b0..0f6ef31 100644
--- a/include/configs/at91sam9260ek.h
+++ b/include/configs/at91sam9260ek.h
@@ -48,6 +48,8 @@
 #define CONFIG_CMD_BOOTZ
 #define CONFIG_OF_LIBFDT
 
+#define CONFIG_SYS_GENERIC_BOARD
+
 /* general purpose I/O */
 #define CONFIG_ATMEL_LEGACY		/* required until (g)pio is fixed */
 #define CONFIG_AT91_GPIO
-- 
1.9.1

